databricks_cli/unity_catalog/cred_cli.py

```diff
@@ -0,0 +1,278 @@
+@eat_exceptions
+@provide_api_client
+def list_dacs_cli(api_client, metastore_id):
+    """
+    List data access configurations.
+
+    Calls the 'listDataAccessConfigurations' RPC endpoint of the Unity Catalog service.
+    Returns array of DataAccessConfigurations.
+
+    """
+    dacs_json = UnityCatalogApi(api_client).list_dacs(metastore_id)
+    click.echo(mc_pretty_format(dacs_json))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Get data access configuration.')
+@click.option('--metastore-id', required=True, type=MetastoreIdClickType(),
+              help='Unique identifier of the metastore parent of the DAC.')
+@click.option('--dac-id', required=True, type=DacIdClickType(),
+              help='Data access configuration ID.')
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def get_dac_cli(api_client, metastore_id, dac_id):
+    """
+    Get data access configuration details.
+
+    Calls the 'getDataAccessConfiguration' RPC endpoint of the Unity Catalog service.
+    Returns details of the DAC specified by its id (TODO: lookup by DAC name?).
+
+    """
+    dac_json = UnityCatalogApi(api_client).get_dac(metastore_id, dac_id)
+    click.echo(mc_pretty_format(dac_json))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Delete data access configuration.')
+@click.option('--metastore-id', required=True, type=MetastoreIdClickType(),
+              help='Unique identifier of the metastore parent of the DAC.')
+@click.option('--dac-id', required=True, type=DacIdClickType(),
+              help='Data access configuration ID.')
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def delete_dac_cli(api_client, metastore_id, dac_id):
+    """
+    Delete data access configuration details.
+
+    Calls the 'deleteDataAccessConfiguration' RPC endpoint of the Unity Catalog service.
+
+    """
+    UnityCatalogApi(api_client).delete_dac(metastore_id, dac_id)
+
+
+@click.group()
+def storage_credentials_group():  # pragma: no cover
+    pass
+
+
+def register_cred_commands(cmd_group):
+    # Register deprecated "verb-noun" commands for backward compatibility.
+    cmd_group.add_command(hide(create_credential_cli), name='create-storage-credential')
+    cmd_group.add_command(hide(list_credentials_cli), name='list-storage-credentials')
+    cmd_group.add_command(hide(get_credential_cli), name='get-storage-credential')
+    cmd_group.add_command(hide(update_credential_cli), name='update-storage-credential')
+    cmd_group.add_command(hide(delete_credential_cli), name='delete-storage-credential')
+
+    # DAC cmds: [TO BE DEPRECATED ONCE STORAGE CREDENTIALS ARE FULLY SUPPORTED]
+    cmd_group.add_command(hide(create_dac_cli), name='create-dac')
+    cmd_group.add_command(hide(list_dacs_cli), name='list-dacs')
+    cmd_group.add_command(hide(get_dac_cli), name='get-dac')
+    cmd_group.add_command(hide(delete_dac_cli), name='delete-dac')
```

Comment:
@adamcain-db Can these be removed?

Note to self: if so, then `DacIdClickType` can go away as well.

Reply:
Alright, thank you. Removed them in 71e1cef969cb316da302f3999c464edc09905410.